### PR TITLE
feat: added function to call TitleEscrowCreator Address

### PR DESCRIPTION
**Pull request by justussoh**
This PR adds module to get `TitleEscrowCreator` address constants from token-registry!

### CODE REVIEW

File: src/config/config.ts [hunk 1]
```diff
@@ -2,6 +2,20 @@ export const DOCUMENT_STORE_CREATOR_ROPSTEN = "0x4077534e82C97Be03A07FB10f5c853d
 export const DOCUMENT_STORE_CREATOR_MAINNET = "0x0";
 export const PROXY_FACTORY_ROPSTEN = "0xba2501bf20593f156879c17d38b6c245ca65de80";
 export const PROXY_FACTORY_MAINNET = "0x0";
+export const TITLE_ESCROW_CREATOR_ROPSTEN =
+  process.env.TITLE_ESCROW_CREATOR_ROPSTEN || "0xB0dE5E22bAc12820b6dbF6f63287B1ec44026c83";
+export const TITLE_ESCROW_CREATOR_MAINET =
+  process.env.TITLE_ESCROW_CREATOR_MAINET || "0x907A4D491A09D59Bcb5dC38eeb9d121ac47237F1";
+export const TITLE_ESCROW_CREATOR_RINKEBY =
+  process.env.TITLE_ESCROW_CREATOR_RINKEBY || "0xa51B8dAC076d5aC80507041146AC769542aAe195";
+export const TITLE_ESCROW_CREATOR_LOCAL =
```
Comment (yehjxraymond) [inline src/config/config.ts:11]: Dont put local, there is no such contract.

Comment (yehjxraymond) [inline src/config/config.ts:11]: leave process.env out, this should describe the deployed one and not be ambiguous about it

File: package.json [hunk 2]
```diff
@@ -63,6 +63,7 @@
     "@ls-age/commitlint-circle": "1.0.0",
     "@typechain/ethers-v5": "^1.0.0",
     "@types/jest": "^25.1.4",
+    "@types/node": "^14.11.7",
```
Comment (yehjxraymond) [inline package.json:66]: can rm if not using env right

Comment (justussoh) [reply package.json:66]: yup!

**Review by yehjxraymond — APPROVED** [review]
lgtm

**Pull request merged**